openwisp_monitoring/monitoring/models.py

```diff
@@ -192,35 +190,30 @@ def read(self, since=None, limit=1, order=None, extra_fields=None):
             q = '{0} LIMIT {1}'.format(q, limit)
         return list(query(q, epoch='s').get_points())
 
-    def _notify_users(self, level, verb, threshold):
+    def _notify_users(self, notification_type, threshold):
         """ creates notifications for users """
-        opts = dict(sender=self, level=level, verb=verb, action_object=threshold)
+        opts = dict(sender=self, type=notification_type, action_object=threshold)
         if self.content_object is not None:
             opts['target'] = self.content_object
         self._set_extra_notification_opts(opts)
         notify.send(**opts)
 
     def _set_extra_notification_opts(self, opts):
-        verb = opts['verb']
         target = opts.get('target')
         metric = str(self).capitalize()
+        opts['metric'] = metric
         status = 'PROBLEM' if not self.is_healthy else 'RECOVERY'
```

Comment:
this doesn't look like is being used, I don't see `PROBLEM` nor `RECOVERY` in the email subject